core-kotlin-modules/core-kotlin-strings-3/src/test/kotlin/com/baeldung/string_extension_functions/StringExtensionFunctionClassUnitTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.baeldung.string_extension_functions
+
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+internal class StringExtensionFunctionClassUnitTest {
+
+
+    @Test
+    fun `using replace() method`(){
+        val inputString0 = "Jelly"
+        val inputString1 = "Kotlin Replace Program"
+        assertEquals(inputString0.replace( 'l', 'z'), "Jezzy")
+        assertEquals(inputString0.replace('j', 'P', true), "Pelly")
+        assertEquals(inputString0.replace('j', 'P'), "Jelly")
+        assertEquals(inputString1.replace("PROGRAM", "Examples", true), "Kotlin Replace Examples")
```

Comment:
One these and all other assertEquals calls in this file, the order of arguments need to be swapped, so that you end up with assertEquals(expected, actual).